watertap/unit_models/zero_order/electrocoagulation_zo.py

```diff
@@ -333,64 +359,73 @@ def eq_cell_voltage(b):
                 == b.overpotential + b.applied_current * b.ohmic_resistance
             )
 
-        @self.Constraint(doc="Area per electrode")
-        def eq_electrode_area_per(b):
-            return b.electrode_area_per == b.electrode_area_total / (
-                b.number_electrode_pairs * 2
-            )
-
-        @self.Constraint(doc="Electrode width")
-        def eq_electrode_width(b):
-            return b.electrode_width == (2 * b.electrode_area_per) ** 0.5
-
-        @self.Constraint(doc="Electrode height")
-        def eq_electrode_height(b):
-            return b.electrode_height == b.electrode_area_per / b.electrode_width
-
         @self.Constraint(doc="Electrode volume")
-        def eq_electrode_volume_per(b):
+        def eq_electrode_volume(b):
             return (
-                b.electrode_volume_per
-                == b.electrode_width * b.electrode_height * b.electrode_thick
+                b.electrode_volume
+                == (b.anode_area + b.cathode_area) * b.electrode_thick
             )
 
+        @self.Constraint(doc="Cathode and anode areas are equal")
```

Comment:
Feels odd to give the option to explicitly define cathode and anode areas, only to be able to fix one or the other. Also, I don't think that the areas actually have to be equal? Maybe better to just omit the constraint and add a degree of freedom to specify both areas.

Reply:
True that the areas don't have to be equal, but given this is a ZO model I think it's best to keep them equal. Also I think we want to avoid leaving DOFs for a ZO model but could add additional complexity for the future detailed model. 

Defining both the cathode and anode areas is more for naming convention and to avoid the ambiguity of the previous model that just had ``electrode_area_per`` which got confusing when formulating other constraints.